### ISSUE TITLE
Askama 0.5 and nom don't seem to work together

### ISSUE THREAD

**Issue by Fiedzia** [djc/askama#75]
'''
error[E0308]: mismatched types
   --> /home/mdziardziel/.cargo/registry/src/github.com-1ecc6299db9ec823/askama_shared-0.5.0/src/parser.rs:102:34
    |
102 |         End(0) => IResult::Error(nom::ErrorKind::Custom(0)),
    |                                  ^^^^^^^^^^^^^^^^^^^^^^^^^
    |                                  |
    |                                  expected enum `nom::Err`, found enum `nom::ErrorKind`
    |                                  help: try using a variant of the expected type: `nom::Err::Code(nom::ErrorKind::Custom(0))`
    |
    = note: expected type `nom::Err<&[u8]>`
               found type `nom::ErrorKind<{integer}>`

error[E0308]: mismatched types
   --> /home/mdziardziel/.cargo/registry/src/github.com-1ecc6299db9ec823/askama_shared-0.5.0/src/parser.rs:109:31
    |
109 |         return IResult::Error(nom::ErrorKind::Custom(0));
    |                               ^^^^^^^^^^^^^^^^^^^^^^^^^
    |                               |
    |                               expected enum `nom::Err`, found enum `nom::ErrorKind`
    |                               help: try using a variant of the expected type: `nom::Err::Code(nom::ErrorKind::Custom(0))`
    |
    = note: expected type `nom::Err<&[u8]>`
               found type `nom::ErrorKind<{integer}>`
'''

**Comment by Fiedzia** [comment]
Following the advice fixes the issue

**Comment by djc** [comment]
Yeah, you should probably use Askama master for now. I'll try to put out a new release soon.
